src/status/chat.nim

```diff
@@ -55,12 +47,27 @@ proc saveChat*(chatId: string, oneToOne = false) =
     }
   ])
 
+proc inactivateChat*(chatId: string) =
```

Comment:
I think the word you were looking for is `deactivate` (I searched, inactivate exists too, just not as used and deactivate fits the context more)